src/lib/jsonParseV2/parseStringVector.cpp

```diff
@@ -68,6 +69,12 @@ bool parseStringVector
 
     std::string value = iter->GetString();
 
+    if (forbiddenIdCharsV2(value.c_str()))
+    {
+      *errorStringP = "forbidden characters in attribute name";
```

Comment:
PR should include a .test covering the case. I mean, a .test with just one step, sending a registration with forbbiden char in attribute name and checking that the corresponding 400 Bad Request is responsed.